core/src/main/resources/org/springframework/ldap/config/spring-ldap-2.0.xsd

```diff
@@ -262,6 +262,209 @@
         </xs:attribute>
     </xs:attributeGroup>
 
+    <xs:attributeGroup name="pooling2.attlist">
```

Comment:
My initial impression is that we do not want to replicate the schema for pool and pool2. Instead I think it might be appropriate to use the namespace support to transparently update from pool to pool2 if only pool2 is found on the classpath. Did you have a specific reason that you wanted to provide specific XML configuration for version 2?